README.md

```diff
@@ -1054,6 +1054,7 @@ Table of Contents
   * [huaweicloud.com](https://www.huaweicloud.com/intl/en-us/product/dns.html) – Free DNS hosting by Huawei
   * [Hetzner](https://www.hetzner.com/dns-console) – Free DNS hosting from Hetzner with API support.
   * [Glauca](https://docs.glauca.digital/hexdns/) – Free DNS hosting for up to 3 domains and DNSSEC support
+  * [Nova53](https://www.nova53.net) - Free Anycast DNS hosting for up to 3 domains, DNSSEC support, Master Slave and Native support.
```

Comment:
Not sure if it’s just me but their site seems a bit broken. Doesn’t load completely, sig up link just goes nowhere etc. 

Also best to avoid the master and slave terminology - rather use what’s on their site.